services/web-server/src/login/strategies/github.js

```diff
@@ -129,9 +131,16 @@ module.exports = class Github {
             userId: profile.id,
             accessToken,
           }, true);
-          const user = await this.getUser({ username: profile.username, userId: Number(profile.id) });
-
-          if (!user) {
+          const [userErr, user] = await tryCatch(
+            this.getUser({ username: profile.username, userId: Number(profile.id) })
+          );
+
+          if (userErr || !user) {
```

Comment:
I think we just want `if (userErr)` here -- the `!user` case is "normal operation"